### DISCUSSION TITLE
Draft Pull Requests not available?

### DISCUSSION THREAD

**Discussion by mdwagner** [community/community#23478] [New to GitHub]
<p>On my organizations github account, when I try to open a new draft pull request (recently released), I’m not given any other option than the normal pull request button. But when I try the same thing on my personal github repos, I am given that option for the draft pull request. So, I’m wondering if it’s something that needs to be enabled on my organizations settings somewhere? Or maybe it’s not available on our Github plan? Thanks.</p>

**Comment by solidstate-tech** [comment]
<p>Happens in my org as well.</p>

**Comment by markalanevans** [comment]
<p>Not available for us either. Any updates?</p>

**Comment by andyblack19** [comment]
<p>Happening in my organisation repos too… my personal private repos do have the option.</p>
<p>Really wanted to use this today to present a technical investigation</p>

**Comment by PlugaruT** [comment]
<p>Yep, in my organisation is not available too.</p>

**Comment by KexyBiscuit** [comment]
<p>I was able to create a draft several hours ago, but now I can’t create neither mark existing ones as ready for review. Seems it’s no longer included in private repo with GitHub Free? WHY CANNOT I MARK IT AS READY FOR REVIEW? I’M BLOCKED BY THIS!</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi, <a class="mention" href="/mdwagner">@mdwagner</a> and everyone else who is having trouble figuring out when this feature is available. Thanks for hanging in there, I appreciate your patience.</p>
<p><a href="https://help.github.com/articles/about-pull-requests/#draft-pull-requests" rel="nofollow noopener">Draft pull requests</a> are available in public repositories with GitHub Free and GitHub Pro, and in public and private repositories with GitHub Team, GitHub Enterprise Cloud, and GitHub Enterprise Server. If you are not seeing this feature, it is probably because the repository that you’re working in does not meet that criteria. If you believe that your repository does meet these criteria and you’re still not seeing the feature, please <a href="https://github.com/contact" rel="nofollow noopener">contact GitHub Support directly</a> so that we can investigate.</p>
<p>Thanks!</p>

**Reply by rohit-dev01** [reply]
Thanks @ThirdEyeRose

**Reply by hira-iqbal** [reply]
I can see this feature in my org repo but not able to apply it due to access issues even when  I have admin access.

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/kexybiscuit">@kexybiscuit</a>,</p>
<p>Sorry about that. Sounds like you ran into a bug that we’ll have to troubleshoot individually. Could you please <a href="https://github.com/contact" rel="nofollow noopener">reach out to GitHub Support directly</a> so that we can work with you on resolving this?</p>
<p>Thanks! </p>

**Comment by mdwagner** [comment]
<p>Thanks for the response!</p>
<p>We have Github Gold, so I’m assuming since that’s not in the list, it’s not supported. Is there any chance it will be added in the future?</p>

**Comment by BBI-YggyKing** [comment]
<p>We’re in the same boat as <a class="mention" href="/mdwagner">@mdwagner</a> - also on GitHub Gold and inerested in draft pull requests.</p>

**Comment by andyblack19** [comment]
<p>Why has such a restriction been imposed?</p>

**Comment by bryantbiggs** [comment]
<p>Incentive to switch off legacy plans most likely. Legacy plans you pay for the repos, current plans you pay for the number of users. Obviously in terms of scaling and income generation, there will be more users than repos in organization accounts and the new plans are priced accordingly. We are on Github Silver which costs us $50 a month and have 38 members in our org. If we switched to a new plan, the monthly cost goes up to $322 a month and will keep going up as our team grows. </p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/mdwagner">@mdwagner</a>, <a class="mention" href="/bbi-yggy">@bbi-yggy</a>, <a class="mention" href="/andyblack19">@andyblack19</a>, and <a class="mention" href="/bryantbiggs">@bryantbiggs</a>,</p>
<p>You are correct that legacy plans were not included in the original plans that draft pull requests were available for. However, we have recently changed this so that draft pull requests are available for <strong>pulic repositories in legacy plans</strong>. Draft pull requests are still unavailable for private repositories in legacy plans.</p>
<p>Hope that helps!</p>

**Reply by Norman-else** [reply]
May I ask when will this feature be enabled for private repositories?

**Comment by BBI-YggyKing** [comment]
<blockquote>
<p><a class="mention" href="/that-pat">@that-pat</a> wrote:</p>
</blockquote>
<blockquote>
<p>we have recently changed this so that draft pull requests are available for **pulic repositories in legacy plans**</p>
</blockquote>
<p>Cool, thanks for letting me know. I’m only interested in this feature for our private repos, so while this is a positive change it doesn’t help my team, but thanks for the update.</p>

**Comment by arturtr** [comment]
<p>It is just a business. Want more? Pay more! But yes, no one wants to pay more.</p>
<p>The situation is the same as it was with Windows XP or old Apple upgradable devices.</p>

**Comment by andyblack19** [comment]
<p><a class="mention" href="/that-pat">@that-pat</a> </p>
<p>Why are you imposing such a restriction?</p>
<p>I’m sure my organisation still pay a lot of money for the subscription per year, albeit you have classed it as a ‘legacy’ plan. Is it to encourage migration to your new, more expensive plans?</p>

**Comment by andyblack19** [comment]
<blockquote>
<p><a class="mention" href="/arturtr">@arturtr</a> wrote:</p>
</blockquote>
<blockquote>
<p>It is just a business. Want more? Pay more! But yes, no one wants to pay more.</p>
</blockquote>
<blockquote>
<p> </p>
</blockquote>
<blockquote>
<p>The situation is the same as it was with Windows XP or old Apple upgradable devices.</p>
</blockquote>
<p>But it’s free to use, on public repos, for free</p>

**Comment by donatj** [comment]
<p>I second the <em>why</em> on this. It seems like a really sketchy move.</p>

**Comment by chuckd** [comment]
<p>I just had a colleague successfully open a Draft PR on my private Github Pro repository, so perhaps they’ve reversed stance on this?! Or just a bug? Docs still say it’s for public repos only for Pro accounts.</p>

**Comment by drdamour** [comment]
<p>it’s not a feature of enterprise server per docs as of 2.16  hoping it shows up for 2.17</p>

**Comment by Norman-else** [comment]
Why not enable this feature for organization private repositories? we pay a lot every year.

**Comment by JaredEzz** [comment]
It's not available for our organization though I believe we fit the criteria for it

**Comment by DigiLiviu** [comment]
I'm sorry why is this not available? You mean we pay for an account and get limited functionality???

**Comment by SinaKarimi7** [comment]
I can't believe that the draft is a premium feature! My plan was to switch to Github from Gitlab, but I am disappointed to find such an obvious feature missing. Due to this simple feature and [this](https://github.com/orgs/community/discussions/12290#discussioncomment-4799358), Gitlab proved to be a better choice...

**Comment by nathany** [comment]
> To access draft pull requests in a private repository, you’ll need to move it to a GitHub Team organization. 

Draft pull requests on private repositories aren't available to individuals, even with a GitHub Pro memberships?! Which are the same monthly price as moving all the repositories to a single-user organization with a GitHub Team membership. I'm really curious why this is the case? 🤦🏼‍♂️

Drafts are a useful feature, even when working solo. It's nice to flag a pull request to myself as incomplete.

If desired, it may even be useful to save some CI hours (and therefore energy and the planet) by not running CI until a draft is ready. That seems like something everyone should have the option of doing!

Until then, I guess I'll go back to using `[WIP]` in the title for personal projects.

**Reply by plutack** [reply]
I wonder as to what could be the motivation to strip this subset of repo or rather not allow repos that fall under this category this seemingly iseful feature.

**Reply by donatj** [reply]
I mean it's just Microsoft trying to differentiate plans by taking away otherwise useful features, rather than actually adding anything of value.

**Comment by nathany** [comment]
FYI https://github.blog/changelog/2025-05-01-draft-pull-requests-are-now-available-in-all-repositories/
